test/test_github.py

```diff
@@ -60,6 +60,26 @@ def test_search_query_upload(self):
         )
         self.assertEqual(datadict['data']['sha'], 'sha-abcdef')
 
+    # test deletion of session annotations that have been deleted in Github repo (e.g. by another user)
+    keep_annotation = { "name": "annotation_to_keep", "filename": "http://test/annotation_to_keep" }
+    delete_annotation =   { "name": "annotation_to_delete", "filename": "http://test/annotation_to_delete" }
+
+    mock_raw_request = MagicMock()
```

Comment:
drat, I meant to change the name ```mock_raw_request``` to ```mock_github_get```. I'll do that along with whatever other changes are still needed.